plugins/foreman_discovery/2.0/index.md

```diff
@@ -23,11 +23,25 @@ direct access both to the provisioning network and Foreman instance:
     --
 
 Foreman Discovery does support "legacy" mode where communication is direct
-with no Foreman Proxy involved. See remarks in this documentation about how to
-configure this mode which is not recommended.
+with no Foreman Proxy involved. Currently (Foreman Discovery 2.0) this is
+still the recommended way to configure it due to technical limitations of
+Foreman Proxy 1.7 (this is explained below).
 
 ## 1.1 Release Notes
 
+For each Foreman version there is particular version of Foreman Discovery
+plugin:
+
+| Foreman  | Plugin  | Image
```

Comment:
Formatting on this table is very plain... maybe check what we do in the main manual to make it clearer?